e2e/TestConstants.ts

```diff
@@ -147,5 +147,15 @@ export const TestConstants = {
     /**
      * Update Channel name on 'Create Operator Subscription' page on OCP, "nightly" or "stable".
      */
-    TS_OCP_UPDATE_CHANNEL_OPERATOR: process.env.TS_OCP_UPDATE_CHANNEL_OPERATOR || 'nightly'
+    TS_OCP_UPDATE_CHANNEL_OPERATOR: process.env.TS_OCP_UPDATE_CHANNEL_OPERATOR || 'nightly',
+
+    /**
+     * Run tests inside of docker container.
+     */
+    TS_SELENIUM_REMOTE_DRIVER: process.env.TS_SELENIUM_REMOTE_DRIVER === 'true',
```

Comment:
Do we need two variables for this? I would say, if TS_SELENIUM_DRIVER_SERVER_URL is empty, use tests-provided webdriver, otherwise use remote one.